frappe/public/js/frappe/utils/datetime.js

```diff
@@ -7,6 +7,7 @@ frappe.defaultDateFormat = "YYYY-MM-DD";
 frappe.defaultTimeFormat = "HH:mm:ss";
 frappe.defaultDatetimeFormat = frappe.defaultDateFormat + " " + frappe.defaultTimeFormat;
 moment.defaultFormat = frappe.defaultDateFormat;
+frappe.defaultUserTZ = Intl.DateTimeFormat().resolvedOptions().timeZone
```

Comment:
A better name would be _userTimezone_, the "default" part of this has no meaning here.